test/system/500-networking.bats

```diff
@@ -626,19 +627,26 @@ load helpers.network
 
         # Verify http contents again: curl from localhost
         # Use retry since it can take a moment until the new container is ready
-        run curl --retry 2 -s $SERVER/index.txt
-        is "$output" "$random_1" "curl 127.0.0.1:/index.txt after auto restart"
-
-        run_podman 0+w restart $cid
+        local curlcmd="curl --retry 2 --retry-connrefused -s $SERVER/index.txt"
```

Comment:
Why not `--retry 3` for consistency?

Reply:
Because the order I did this in was, first did the `--retry 3` in `infra_container_test.go`, picking `3` just out of habit, and then I made a giant sweep through the rest of the code, saw this "2", and left it as "2" to minimize diffs. An argument could be made that I should change my initial `3` to `2`, to be consistent with already-existing code...

Reply:
Update: left this one as-is, updated the new one (at top) to 2.